ports/espressif/supervisor/port.c

```diff
@@ -238,20 +247,6 @@ safe_mode_t port_init(void) {
     common_hal_never_reset_pin(&pin_GPIOn_EXPAND(CONFIG_CONSOLE_UART_RX_GPIO));
     #endif
 
-    #if DEBUG
-    // debug UART
-    #if defined(CONFIG_IDF_TARGET_ESP32C3)
-    common_hal_never_reset_pin(&pin_GPIO20);
-    common_hal_never_reset_pin(&pin_GPIO21);
-    #elif defined(CONFIG_IDF_TARGET_ESP32C6)
-    common_hal_never_reset_pin(&pin_GPIO16);
-    common_hal_never_reset_pin(&pin_GPIO17);
-    #elif defined(CONFIG_IDF_TARGET_ESP32S2) || defined(CONFIG_IDF_TARGET_ESP32S3)
-    common_hal_never_reset_pin(&pin_GPIO43);
-    common_hal_never_reset_pin(&pin_GPIO44);
-    #endif
-    #endif
-
```

Comment:
Are these now moot never-resets due to something else?

Reply:
Yup! The reset forbidden stuff covers it: https://github.com/adafruit/circuitpython/blob/main/ports/espressif/common-hal/microcontroller/Pin.c#L122